packages/react/src/editor/styles.css

```diff
@@ -1,144 +1,186 @@
-/* Default theme params, overwritten by BlockNoteView */
-:root {
-    --bn-color-editor-text: #3F3F3F;
-    --bn-color-editor-background: #FFFFFF;
-    --bn-color-menu-text: #3F3F3F;
-    --bn-color-menu-background: #FFFFFF;
-    --bn-color-tooltip-text: #3F3F3F;
-    --bn-color-tooltip-background: #EFEFEF;
-    --bn-color-hovered-text: #3F3F3F;
-    --bn-color-hovered-background: #EFEFEF;
-    --bn-color-selected-text: #FFFFFF;
-    --bn-color-selected-background: #3F3F3F;
-    --bn-color-disabled-text: #AFAFAF;
-    --bn-color-disabled-background: #EFEFEF;
-
-    --bn-color-shadow: #CFCFCF;
-    --bn-shadow-medium: 0 4px 12px var(--bn-color-shadow);
-
-    --bn-color-border: #EFEFEF;
-    --bn-border: 1px solid var(--bn-color-border);
-    --bn-shadow-light: 0 2px 6px var(--bn-color-border);
-
-    --bn-color-side-menu: #CFCFCF;
-
-    --bn-color-highlight-text-gray: #9b9a97;
-    --bn-color-highlight-background-gray: #ebeced;
-    --bn-color-highlight-text-brown: #64473a;
-    --bn-color-highlight-background-brown: #e9e5e3;
-    --bn-color-highlight-text-red: #e03e3e;
-    --bn-color-highlight-background-red: #fbe4e4;
-    --bn-color-highlight-text-orange: #d9730d;
-    --bn-color-highlight-background-orange: #f6e9d9;
-    --bn-color-highlight-text-yellow: #dfab01;
-    --bn-color-highlight-background-yellow: #fbf3db;
-    --bn-color-highlight-text-green: #4d6461;
-    --bn-color-highlight-background-green: #ddedea;
-    --bn-color-highlight-text-blue: #0b6e99;
-    --bn-color-highlight-background-blue: #ddebf1;
-    --bn-color-highlight-text-purple: #6940a5;
-    --bn-color-highlight-background-purple: #eae4f2;
-    --bn-color-highlight-text-pink: #ad1a72;
-    --bn-color-highlight-background-pink: #f4dfeb;
+@import url("@mantine/core/styles.css");
+/* TODO: Cannot resolve file? */
+@import url("@blocknote/core/style.css");
+
+/* Default theme params */
+.bn-editor-wrapper {
```

Comment:
Instead of `.bn-editor-wrapper` I think I'd prefer `.bn-editor` or `.bn-container` or sth